runtime/src/main/java/io/micronaut/health/HeartbeatTask.java

```diff
@@ -37,17 +37,14 @@
     private AtomicReference<ServiceInstance> eventReference = new AtomicReference<>();
 
     private final ApplicationEventPublisher eventPublisher;
-    private final HeartbeatConfiguration configuration;
     private final CurrentHealthStatus currentHealthStatus;
 
     /**
      * @param eventPublisher      To publish the events related to heartbeat
-     * @param configuration       The configurations for heartbeat
      * @param currentHealthStatus The current status of health indicator
      */
-    public HeartbeatTask(ApplicationEventPublisher eventPublisher, HeartbeatConfiguration configuration, CurrentHealthStatus currentHealthStatus) {
+    public HeartbeatTask(ApplicationEventPublisher eventPublisher, CurrentHealthStatus currentHealthStatus) {
```

Comment:
This is a breaking change and must be reverted

Reply:
Argument restored.